edit/buildozer.go

```diff
@@ -417,6 +417,46 @@ func cmdRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
 	return nil, nil
 }
 
+func cmdRemoveIfEqual(opts *Options, env CmdEnvironment) (*build.File, error) {
+	attr := env.Args[0]
+	args := env.Args[1:]
+	expr := getAttrValueExpr(attr, args, env)
```

Comment:
getAttrValueExpr is using some heuristics to guess the expected type of an attribute. It is a bit brittle and I think we don't need this function here.

e.g. If you call `env.Rule.Attr` and get a string expression, you can just compare the content of the string with `attr`. No need to build an expr.

Reply:
done